starlite/openapi/controller.py

```diff
@@ -58,13 +59,40 @@ def get_schema_from_request(request: Request) -> "OpenAPI":
             An [OpenAPI][pydantic_openapi_schema.v3_1_0.open_api.OpenAPI] instance.
 
         Raises:
-            [ImproperlyConfiguredException][starlite.exceptions.ImproperlyConfiguredException]: If the application
-                `openapi_schema` attribute is `None`.
+            ImproperlyConfiguredException: If the application `openapi_config` attribute is `None`.
         """
         if not request.app.openapi_schema:  # pragma: no cover
             raise ImproperlyConfiguredException("Starlite has not been instantiated with OpenAPIConfig")
         return request.app.openapi_schema
 
+    def should_serve_endpoint(self, request: Request) -> bool:
+        """This method verifies that the requested path is within the enabled
+        endpoints in the openapi_config.
+
+        Args:
+            request: To be tested if endpoint enabled.
+
+        Returns:
+            A boolean.
+
+        Raises:
+            ImproperlyConfiguredException: If the application `openapi_config` attribute is `None`.
```

Comment:
Same as previously, for the linkage:

```suggestion
            [ImproperlyConfiguredException][starlite.exceptions.ImproperlyConfiguredException]: If the application `openapi_config` attribute is `None`.
```